torch/csrc/jit/ivalue.h

```diff
@@ -17,8 +17,8 @@ struct TORCH_API ConstantString : c10::intrusive_ptr_target {
  private:
   const std::string str_;
  public:
-  ConstantString(const std::string & str)
-  : str_(str) {}
+  ConstantString(std::string str)
```

Comment:
Why is not passing a constant reference better?

Reply:
Best explained here:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html